README.md

```diff
@@ -93,32 +50,54 @@ In order to allow the assistant user to create location reservations and service
     
 
-### Part 3 - Request the Assistant Integration ID
+### Part 2 - Submit the Provisioning Request and Check the results
+
+#### E) COMPLETE THE PROVISIONING FORM
+
+The OM package imported contains a form that will execute a process of gathering data for the Assistant and providing that data to IBM.  The form asks for the user name and password for the account created in an earlier step.  This is needed so the IBM's TRIRIGA Assistant services can make an OSLC call back into your TRIRIGA instance and gather building and room names.
 
-#### F) REQUEST YOUR ASSISTANT INTEGRATION ID.
+1. From the TRIRIGA Main Page, click on Requests > Request IBM TRIRIGA Assistant  (if you do not see this menu item, check the trouble shooting section below.)
+2. Enter the URL for your TRIRIGA instance. Note: The instance must be accessible without a VPN and through HTTPS. Include the complete path, starting with `https://`.
+3. Enter the user name and password for the assistant account you created earlier.
+4. Enter your name and email address so we can contact you when the assistant services have been provisioned for your instance.
+5. Click Submit.
+Note: You won't get any direct feedback on the form if everything went well or if there were failures.
 
-Your integration ID will be used to connect your TRIRIGA with the Assistant Skill on the cloud. You will use this Integration ID in the next step.
+#### F) CHECK IF INFORMATION WAS RECEIVED CORRECTLY
 
-To request your Integration ID, send the following to **[jtmoore@us.ibm.com](jtmoore@us.ibm.com)**:
-1. The Excel files generated from the three reports:
-- Reservable Spaces (Step A).
-- All Active Locations (Step A).
-- Location Timezones (Step B).
-2. The URL for your TRIRIGA instance. Note: The instance must be accessible without a VPN and through HTTPS. 
-3. The username and password for the assistant account (Step D).
+To check that the information was sent correctly to the IBM TRIRIGA Assistant services, perform the following steps.
 
+1. From the TRIRIGA Main Page, click on Tools > System Setup > Integration Object.
+2. Search for `ibmTriAssistant` in the Name column of the Integration Objects table.
+3. Click on the `ibmTriAssistant - POST - tririgaOrchestratorCF` integration object.
+4. Check the value for `Status`.  It should be `Ready`.  If `Failed`, then please send your IBM representative the information found in the `Integration Summary` field from the most recent execution in the `Execute History` table.
 
-### Part 4 - Add Assistant UI to Workplace Services apps
+#### G) WAIT FOR INTEGRATION ID FROM IBM
 
-#### G) EDIT THE WORKPLACE SERVICES APP TO ADD CHAT UI ACCESS.
+Once the information has been successfully received by the IBM TRIRIGA Assistant services, you will be contacted through email at the email address provided on the form.  Once you receive the Integration ID, you may proceed with the rest of the steps.
+
```

Comment:
Might be good to add a failsafe here, like `If you do not receive your integration ID within 2 business days please feel free to contact support at [Salesforce Link].

Reply:
are you getting me a link for "Salesforce Link".  I didn't know if you were and I don't know it so I was just going to say "contact IBM representative".  @JordanTMoore

Reply:
I don't have the access to grab in from the environment. I'll put in the one team request for that. If you can't see it that should work for now.

Reply:
added failsafe.  If they don't hear something after a week, then contact an IBM rep.